app/components/footer.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <footer className="bg-gray-100 text-black py-4 border-t">
+      <div className="container mx-auto flex justify-between items-center">
+        <p className="text-sm">
+          &copy; 2025 CodeCrafters Inc. | Privatlivspolitik | Copyright
```

Comment:
If you don't want to manually update year every year, then usually you generate it through javascript, fx `new Date().getUTCFullYear()`

`Privatlivspolitik` usually a link, isn't it? Or it'll be updated later?